Makefile

```diff
@@ -1,22 +1,59 @@
+ARCH    ?= amd64
+ALL_ARCH = amd64 arm arm64 ppc64le s390x
+
 .PHONY: all test
 
 IMAGE_BUILD_CMD := docker build
 
 VERSION := $(shell git describe --tags --dirty --always)
 
+IMAGEARCH ?=
+QEMUARCH ?= x86_64
```

Comment:
Even though probably not a big problem in reality, I would put these inside an `ifeq` block, too. Otherwise you will happily get an (x86_64) image when running something like `make ARCH=mips64`.

Reply:
Done.

Reply:
Done.